packages/ra-core/src/controller/create/useCreateController.security.stories.tsx

```diff
@@ -0,0 +1,234 @@
+            { id: 3, title: 'Post #3', votes: 30 },
+            { id: 4, title: 'Post #4', votes: 40 },
+            { id: 5, title: 'Post #5', votes: 50 },
+            { id: 6, title: 'Post #6', votes: 60 },
+            { id: 7, title: 'Post #7', votes: 70 },
+        ],
+    },
+    process.env.NODE_ENV === 'development'
+);
+
+const Post = (props: Partial<CreateControllerProps>) => {
+    const params = useCreateController({
+        resource: 'posts',
+        ...props,
+    });
+    return (
+        <div style={styles.mainContainer}>
+            {params.isPending ? <p>Loading...</p> : <div>Create view</div>}
+        </div>
+    );
+};
+
+const defaultAuthProvider: AuthProvider = {
+    checkAuth: () => new Promise(resolve => setTimeout(resolve, 500)),
+    login: () => Promise.resolve(),
+    logout: () => Promise.resolve(),
+    checkError: () => Promise.resolve(),
+    getPermissions: () => Promise.resolve(),
+};
+
+export const Authenticated = ({
+    authProvider = defaultAuthProvider,
+    dataProvider = defaultDataProvider,
+}: {
+    authProvider?: AuthProvider;
+    dataProvider?: DataProvider;
+}) => {
+    return (
+        <TestMemoryRouter initialEntries={['/posts/create']}>
+            <CoreAdminContext
+                dataProvider={dataProvider}
+                authProvider={authProvider}
+            >
+                <CoreAdminUI>
+                    <Resource name="posts" create={Post} />
+                </CoreAdminUI>
+            </CoreAdminContext>
+        </TestMemoryRouter>
+    );
+};
+
+export const DisableAuthentication = ({
+    authProvider = defaultAuthProvider,
+    dataProvider = defaultDataProvider,
+}: {
+    authProvider?: AuthProvider;
+    dataProvider?: DataProvider;
+}) => {
+    return (
+        <TestMemoryRouter initialEntries={['/posts/create']}>
+            <CoreAdminContext
+                dataProvider={dataProvider}
+                authProvider={authProvider}
+            >
+                <CoreAdminUI>
+                    <Resource
+                        name="posts"
+                        create={<Post disableAuthentication />}
+                    />
+                </CoreAdminUI>
+            </CoreAdminContext>
+        </TestMemoryRouter>
+    );
+};
+
+export const CanAccess = ({
+    authProviderDelay = 300,
+}: {
+    authProviderDelay?: number;
+}) => {
+    return (
+        <TestMemoryRouter initialEntries={['/posts']}>
+            <AccessControlAdmin
+                authProviderDelay={authProviderDelay}
+                queryClient={new QueryClient()}
+            />
+        </TestMemoryRouter>
+    );
+};
+
+const AccessControlAdmin = ({
+    authProviderDelay,
+    queryClient,
+}: {
+    authProviderDelay?: number;
+    queryClient: QueryClient;
+}) => {
+    const [authorizedResources, setAuthorizedResources] = React.useState({
+        'posts.list': true,
+        'posts.create': true,
+        'posts.edit': false,
+        'posts.show': false,
+    });
```

Comment:
this seems overkill as you only ever modify the `create` permission

Reply:
Keeping at least the list makes testing manually and automatically by navigating easier